baselines/fedprox/pyproject.toml

```diff
@@ -141,3 +114,59 @@ exclude = [
 
 [tool.ruff.pydocstyle]
 convention = "numpy"
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.flwr.app]
+publisher = "itsmaddox-j"
+
+[tool.flwr.app.components]
+serverapp = "fedprox.server_app:app"
+clientapp = "fedprox.client_app:app"
+
+[tool.flwr.app.config]
+[tool.flwr.app.config.algorithm]
+name = "FedProx"
+num_server_rounds = 100
+fraction_fit = 0.01
+fraction_evaluate= 0.0
+min_evaluate_clients= 0
+min_available_clients= 1000
+min_fit_clients= 1000
+local_epochs = 10
+stragglers_fraction = 0.9
+learning_rate = 0.03
+mu = 1.0
+num_clients = 1000
```

Comment:
Let's indicate with a comment that this `num_clients` is here just for the 	`ServerApp` (and that it doesn't have an impact on the size of the federation). My understanding is that you are using this here because the `ServerApp` can't access the number of nodes connected outside the strategy and it can't read the `node_config` either.  This won't be a problem with the upcoming `Message API` 👼